### ISSUE TITLE
Webpack's ProvidePlugin replaces Symbol with es6-symbol, even in polyfill

### ISSUE THREAD

**Issue by jdatskuid** [medikoo/es6-symbol#29]
I would like to use webpack's `ProvidePlugin` to detect uses of `Symbol` throughout my codebase and replace them with the polyfill provided by es6-symbol.

```javascript
const config = {
	plugins: [
		new webpack.ProvidePlugin({
			"Symbol": "es6-symbol",
		})
	],
};
```

However, this causes a problem. es6-symbol references `Symbol` directly rather than `window.Symbol` or `global.Symbol`, so webpack replaces these references with a reference to the es6-symbol polyfill (yes, the polyfill is referencing itself), which makes it look like `Symbol` is not available, even on newer browsers.

**polyfill.js** ends up wrapped in this:
```javascript
/* WEBPACK VAR INJECTION */(function(global, Symbol) {
// ES2015 Symbol polyfill for environments that do not (or partially) support it

var d              = require('d')
  // ...

// This fails because of the Symbol argument in the webpack var injection function
if (typeof Symbol === 'function') {
	NativeSymbol = Symbol;
	try {
		String(NativeSymbol());
		isNativeSafe = true;
	} catch (ignore) {}
}

// ...
/* WEBPACK VAR INJECTION */}.call(this, __webpack_require__(/*! es6-symbol */ "./node_modules/es6-symbol/index.js")))
```

`ProvidePlugin` greatly increases the utility of es6-symbol by reducing the number of required `import` statements and ensuring that a forgotten `import` does not wreck a web app running in IE11. While it may not be part of everyone's workflow, it would be very nice to change a couple lines of code to enable this use-case:

**polyfill.js**
```javascript
// ES2015 Symbol polyfill for environments that do not (or partially) support it

var d              = require('d')
  // ...
  , jsGlobal = (window || global || self || this);

// This could probably be optimized more... just an off-the-cuff suggestion
if (typeof jsGlobal.Symbol === 'function' || typeof Symbol === 'function') {
	NativeSymbol = jsGlobal.Symbol || Symbol;
	// ...
}
```

I am willing to submit a PR if this issue is accepted.

**Comment by medikoo** [comment]
@jdatskuid great thanks for report, that's indeed interesting.

Yes, I'm open for PR. Best if you resolve `global` from `es5-ext/global` (`es5-ext` is already a dependency of a a package)

**Comment by medikoo** [comment]
Thanks. Fixed with #30, and published as v3.1.2
